File: Basics/Conditional Statements Advanced/Exercise/03.py
flower = input()
count_flower = int(input())
budget = int(input())

price = 0
all_cost = 0

if flower == "Roses":
    price = 5
  
    if count_flower > 80:
        all_cost = (count_flower * price) * 0.9
    else:
        all_cost = count_flower * price
      
elif flower == "Dahlias":
    price = 3.80
  
    if count_flower > 90:
        all_cost = (count_flower * price) * 0.85
    else:
        all_cost = count_flower * price
      
elif flower == "Tulips":
    price = 2.80
  
    if count_flower > 80:
        all_cost = (count_flower * price) * 0.85
    else:
        all_cost = count_flower * price
      
elif flower == "Narcissus":
    price = 3
  
    if count_flower < 120:
        all_cost = (count_flower * price) * 1.15
    else:
        all_cost = count_flower * price
      
elif flower == "Gladiolus":
    price = 2.50
  
    if count_flower < 80:
        all_cost = (count_flower * price) * 1.20
    else:
        all_cost = count_flower * price

left_money = abs(budget - all_cost)

if budget >= all_cost:
    print(f"Hey, you have a great garden with {count_flower} "
          f"{flower} and {left_money:.2f} leva left.")
else:
    print(f"Not enough money, you need {left_money:.2f} leva more.")

File: JS-Basics/Conditional Statements Advanced/Exercise/03.js
function flowerShop(flowerType, flowerCount, budget) {
    let pricePerFlower = 0;

    switch (flowerType) {
        case "Roses":
            pricePerFlower = 5;
            break;
        case "Dahlias":
            pricePerFlower = 3.80;
            break;
        case "Tulips":
            pricePerFlower = 2.80;
            break;
        case "Narcissus":
            pricePerFlower = 3;
            break;
        case "Gladiolus":
            pricePerFlower = 2.50;
            break;
    }

    let totalPrice = pricePerFlower * flowerCount;

    if (flowerType === "Roses" && flowerCount > 80) {
        totalPrice *= 0.90;
    } else if (flowerType === "Dahlias" && flowerCount > 90) {
        totalPrice *= 0.85;
    } else if (flowerType === "Tulips" && flowerCount > 80) {
        totalPrice *= 0.85;
    } else if (flowerType === "Narcissus" && flowerCount < 120) {
        totalPrice *= 1.15;
    } else if (flowerType === "Gladiolus" && flowerCount < 80) {
        totalPrice *= 1.20;
    }

    if (budget >= totalPrice) {
        let moneyLeft = (budget - totalPrice).toFixed(2);
        console.log(`Hey, you have a great garden with ${flowerCount} ${flowerType} and ${moneyLeft} leva left.`);
    } else {
        let moneyNeeded = (totalPrice - budget).toFixed(2);
        console.log(`Not enough money, you need ${moneyNeeded} leva more.`);
    }
}
